kubectl-minio/cmd/delete.go

```diff
@@ -71,6 +74,7 @@ func newDeleteCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	cmd = helpers.DisableHelp(cmd)
 	f := cmd.Flags()
 	f.StringVarP(&o.operatorOpts.Namespace, "namespace", "n", helpers.DefaultNamespace, "namespace scope for this request")
+	f.BoolVarP(&o.force, "force", "f", false, "force delete without confirmation")
```

Comment:
```suggestion
	f.BoolVarP(&o.force, "force", "f", false, "allow deleting without confirmation")
```

Reply:
We should also add another flag `--dangerous` since this is not a "reversible" operation